### PR TITLE
Example - Warming - Change image + small fixes

### PR DESCRIPTION
**Pull request by jeannejoachim**
# Description of the problem

- first image had no background and so it was not readable in dark mode
- some small typo here and there

# Description of the solution

- Fixing it all

# Documentation

examples/multiphysics/warming-up-a-viscous-fluid : 
- warming-up-a-viscous-fluid.rst
- images/warming_scheme.pgn

### CODE REVIEW

**Review by lpsaavedra — CHANGES_REQUESTED** [review]
Nice example. However, I think it still needs some work before getting merged:

- Add the sections that are missing: features, location of the example, description, the different sections of the parameter file, running the simulation and possibilities for extensions.
- Add the ^\circ before the C for celsius whenever it is written in math mode (line 219 of the main file)

In addition, I wonder why the multiphysics dam-break-VOF example is there but not added in the multiphysics.rst. Is there any particular reason for that? We may need to create an additional PR to fix this. Also there is a warning regarding a help_from_2D_to_2D file that was added in the cfd.rst but it does not exits. Is this a WIP?

**Comment by blaisb** [comment]
> In addition, I wonder why the multiphysics dam-break-VOF example is there but not added in the multiphysics.rst. Is there any particular reason for that? 

There is no reason for that, it's a mistake in the original PR we did not catch.

**Comment by lpsaavedra** [comment]
> There is no reason for that, it's a mistake in the original PR we did not catch.

OK, then in that case I will create an additional PR to add it

**Review by CaroleAnneDaunais — APPROVED** [review]

File: doc/source/parameters/cfd/cfd.rst
Comment (jeannejoachim) [reply doc/source/parameters/cfd/cfd.rst:28]: it's a WIP in another branch, I don't know that it was added there so I will remove it ^^"

**Comment by jeannejoachim** [comment]
> Nice example. However, I think it still needs some work before getting merged:
> 
>     * Add the sections that are missing: features, location of the example, description, the different sections of the parameter file, running the simulation and possibilities for extensions.
> 
>     * Add the ^\circ before the C for celsius whenever it is written in math mode (line 219 of the main file)

You are right, this PR is a good occasion of reviewing this example and make it in the same fashion as the others! I did not have this in mind (I just wanted to change the first figure), but I can definitely do that now!

**Comment by lpsaavedra** [comment]
Seems good to me now Jeanne! The example looks amazing and it has all the sections that we have in other examples. I will only quickly fix some warnings that were generated and then it should be ready to merge!

File: doc/source/examples/multiphysics/warming-up-a-viscous-fluid/warming-up-a-viscous-fluid.rst
Comment (lpsaavedra) [inline doc/source/examples/multiphysics/warming-up-a-viscous-fluid/warming-up-a-viscous-fluid.rst:330]:
```suggestion
* Test a **different time integration scheme** and see if there is any difference in the computational cost and/or the precision with regards to the analytical solution.
* See how the **resolution order** (``velocity order``, ``pressure order`` and ``temperature order``) affects the precision with regards to the analytical solution.
```

**Review by lpsaavedra — COMMENTED** [review]
Seems good to me now Jeanne! The example looks amazing and it has all the sections that we have in other examples. I will quickly fix some warnings that were generated and then it should be ready to merge after correcting this small typo

File: doc/source/examples/multiphysics/warming-up-a-viscous-fluid/warming-up-a-viscous-fluid.rst
Comment (blaisb) [inline doc/source/examples/multiphysics/warming-up-a-viscous-fluid/warming-up-a-viscous-fluid.rst:58]:
```suggestion
    Heat transfer phenomenon occur at a much larger characteristic time than fluid transport phenomenon. To reach a stable state for the system, the end time much be quite big, but the time step can also be increased (in the limit of numerical stability).
```

File: doc/source/examples/multiphysics/warming-up-a-viscous-fluid/warming-up-a-viscous-fluid.rst
Comment (blaisb) [inline doc/source/examples/multiphysics/warming-up-a-viscous-fluid/warming-up-a-viscous-fluid.rst:117]:
```suggestion
    As the fluid velocity is not influenced by heat transfer (one-way coupling), the fluid velocity will remain constant for the whole simulation across the domain, and as heat transfer occurs at a larger scale, the mesh can be coarse.
```

**Review by lpsaavedra — APPROVED** [review]

**Review by blaisb — APPROVED** [review]
Just two small comments.

**Comment by jeannejoachim** [comment]
Done! Thank you @lpsaavedra for fixing the warnings, I learned something :)

**Pull request merged**
